examples/kubernetes/volume-expansion/README.md

```diff
@@ -0,0 +1,49 @@
+# Volume Expansion
+
+## Context
+
+Users are able to change the capacity of their volumes. 
+Currently, the capacity can ONLY be increased, and NOT decreased.
+Only storage capacity can be adjusted. Throughput can NOT be modified.
+When scaling an FSxZ Volume, both StorageCapacityQuota and StorageCapacityReservation are increased to the same value.
```

Comment:
I think we add a disclaimer for this:
> Storage capacity minimum increase – Each SSD storage capacity increase must be a minimum of 10 percent of the file system's current SSD storage capacity, up to the maximum allowed value of 512 Tebibytes (TiB).

Source: https://docs.aws.amazon.com/fsx/latest/OpenZFSGuide/managing-storage-capacity.html